tests/test_service.py

```diff
@@ -52,18 +80,92 @@ def test_default_error(client, mock_env_access_key):
     expected_msg_error = jsend.error('404 - Not Found')
     assert json.loads(response.content) == expected_msg_error
 
-def test_create_submission(client, mock_env_access_key):
+def test_create_submission(client, queue, mock_env_access_key, mock_external_system_env):
     body = {
         'data': '{"foo":"bar"}'
     }
-    headers = {"Content-Type": "application/x-www-form-urlencoded"}
-    response = client.simulate_post('/submissions', body = urlencode(body), headers = headers)
+
+    with patch('service.resources.jobs.get_queue') as mock_queue:
+        mock_queue.return_value = queue
+
+        with patch('service.resources.jobs.requests.post') as mock_post:
+            mock_post.return_value.status_code = 200
+            mock_post.return_value.text = EXTERNAL_RESPONSE
+            # mock_post.return_value.json.return_value = json.loads(EXTERNAL_RESPONSE)
+            response = client.simulate_post('/submissions', body = urlencode(body), headers = HEADERS)
```

Comment:
This works because of setting the variable auto_parse_form_urlencoded = True.

https://falcon.readthedocs.io/en/stable/api/api.html#falcon.RequestOptions.auto_parse_form_urlencoded